### ISSUE TITLE
Globaleaks 4.9.6 fails due to missing txtorcon.onion 

### ISSUE THREAD

**Issue by anaaviles** [globaleaks/GlobaLeaks#3229]
**Describe the bug**

After globaleaks fails to start after upgrade 4.9.5 to 4.9.6 
OS Ubuntu 18.04.6 LTS

```
May 30 07:36:57 leaks.server globaleaks[1266]:  * Starting ...                                                                                         
May 30 07:36:58 leaks.server globaleaks[1266]: WARNING: The current long term supported platform is Debian 11 (bullseye)                               
May 30 07:36:58 leaks.server globaleaks[1266]: WARNING: It is recommended to use only this platform to ensure stability and security                   
May 30 07:36:58 leaks.server globaleaks[1266]: WARNING: To upgrade your system consult: https://docs.globaleaks.org/en/main/user/admin/Upgra…Guide.html
May 30 07:36:58 leaks.server globaleaks[1266]: Failed to load application: No module named 'txtorcon.onion'                                            
May 30 07:36:58 leaks.server globaleaks[1266]:    ...fail!                                                                                             
May 30 07:36:58 leaks.server systemd[1]: globaleaks.service: Control process exited, code=exited status=1                                              
May 30 07:36:58 leaks.server systemd[1]: globaleaks.service: Failed with result 'exit-code'.                                                           
May 30 07:36:58 leaks.server systemd[1]: Failed to start LSB: Start the GlobaLeaks server..  
```

**Expected behavior**
Starts

**Comment by evilaliv3** [comment]
Thank you @anaaviles 

I confirm that this is a bug related to the bionic packaging that seems to missa more recent component available in `focal` and `jammy`.

We will proceed releasing a patch to get aroung this bug by the end of this morning and update this ticket.

P.s. if you do not have specific reason to still run ubuntu bionic we would advise to update to a recent version (jammy) by using our distribution upgrade instructions: https://docs.globaleaks.org/en/main/user/admin/UpgradeGuide.html#upgrade-of-the-distribution-version

**Comment by evilaliv3** [comment]
Fixed and released in version 4.9.8
